platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+class JSONField(models.TextField):
+    def __init__(self, *args, **kwargs):
```

Comment:
Not sure you'll need to override this, but obviously this class is a WIP. Looks good.